resources/views/livewire/checkout.blade.php

```diff
@@ -70,6 +70,20 @@ class="form-control @error('name') is-invalid @enderror">
                             @enderror
                         </div>
 
+                        <div class="form-group">
```

Comment:
wait ppl choose there own billing cycle? i personally hate this alot and wouldn't want my customer choose there own billing cycles. since i personally dislike changing it from hourly aswel ( i rather had it every minut but this is resource intensive on weaker servers ) 

i'm a fan of the populair cloud services that you get billed as you go, not get 1 huge ass bil at the end